js/packages/cli/src/helpers/instructions.ts

```diff
@@ -74,8 +74,8 @@ export async function createConfigAccount(
     4 +
     configData.maxNumberOfLines.toNumber() * CONFIG_LINE_SIZE +
     4 +
-    Math.ceil(configData.maxNumberOfLines.toNumber() / 8);
-
+    Math.ceil(configData.maxNumberOfLines.toNumber() / 8) +
+    1;
   return anchor.web3.SystemProgram.createAccount({
```

Comment:
Nit: Given the boolean comes earlier in the struct, this 1 should be in the place where it actually takes place

Reply:
We should be good on this.